src/test/tests.ts

```diff
@@ -896,7 +893,15 @@ describe('Server', function () {
       //do nothing
     });
 
-    client.unsubscribe(subId);
+    setTimeout(() => {
+      // TODO
+      // Hagaico I had the need to add this timeout here
```

Comment:
@DxCx If i don't set a timeout of 100ms until I make the unsubscribe, the onUnsubscribe is never called because unsubscribe isn't really executed in the server side, as the operation hasn't registered yet in the server :(

I haven't put unsubscribe inside the callback where 'do nothing' is because this callback isn't called :(